mscore/scoreaccessibility.cpp

```diff
@@ -93,10 +93,13 @@ QString AccessibleScoreView::text(QAccessible::Text t) const
             case QAccessible::Value:
             case QAccessible::Description: {
                   QString msg = s->score()->accessibleMessage();
+                  QString info = s->score()->accessibleInfo();
                   if (msg.isEmpty())
-                        return s->score()->accessibleInfo();
+                        return info;
                   s->score()->setAccessibleMessage(""); // clear the message
-                  return msg;
+                  if (info.isEmpty())
+                        return msg;
+                  return tr("%1, %2").arg(msg).arg(info);
```

Comment:
Can you summarize when you expect this to hold?  Is it to read the mode text and note info?

Reply:
Exactly. Typical values are:
```c++
msg  = "Steptime note input mode";
info = "Note C4 (quarter), beat 3, measure 1, staff 2";
```
`msg` could also relate to an action, such as `"Staccato above added"`, but `info` always describes the current selection.

Reply:
Looking through the code paths, it does seem these would always be distinct - I had worried at first they might be the same.  Now the only thing I'd be concerned about is if there is ever a time when navigating through text (so msg is the character to read) that info would not have already been cleared and would still have the full text.  Not that I know of a case offhand, but that's what I'll still want to test.

Reply:
@MarcSabatella, you were right to be concerned, hence I have explicitly emptied the info string during text editing. See [below](https://github.com/musescore/MuseScore/pull/6210/commits/628aa6f511ec5e647099e3c69083944ccb7cc675#diff-5f8c5742ec536a42250a6de753c09218R305).